libclamunrar/extract.cpp

```diff
@@ -646,9 +646,9 @@ bool CmdExtract::ExtractCurrentFile(Archive &Arc,size_t HeaderSize,bool &Repeat)
       wcsncpyz(DestFileName,Cmd->DllDestName,ASIZE(DestFileName));
 #endif
 
-    if (ExtrFile && Command!='P' && !Cmd->Test && !Cmd->AbsoluteLinks &&
-        ConvertSymlinkPaths)
-      ExtrFile=LinksToDirs(DestFileName,Cmd->ExtrPath,LastCheckedSymlink);
+    // if (ExtrFile && Command!='P' && !Cmd->Test && !Cmd->AbsoluteLinks &&
```

Comment:
Someone once told me, "don't commit commented-out code -- that's what SCM is for"

Reply:
Haha that's a good point.  I say the same thing sometimes. Okay, will just delete it.

Reply:
Fixed.